analysis/check_packages.R

```diff
@@ -5,7 +5,7 @@
 #http://www.vikram-baliga.com/blog/2015/7/19/a-hassle-free-way-to-verify-that-r-packages-are-installed-and-loaded
 
 #add new packages to the chain here
-packages = c("foreign","ggplot2","stargazer")
+packages = c("foreign","ggplot2","stargazer", "readr")
```

Comment:
You need to either load the library in the main script or here. Right now the `organize_data.R` script fails because you call `read_fwf` without first loading the necessary library.